include/miral/miral/external_client.h

```diff
@@ -37,18 +37,16 @@ class ExternalClientLauncher
 
     void operator()(mir::Server& server);
 
-    void launch(std::vector<std::string> const& command_line) const;
+    /// Launch Wayland and X11 support (if enabled).
+    /// \return The pid of the latest process that was launched or -1.
```

Comment:
Under what circumstances will this return `-1`? `launch_app_env` throws if `fork()` returns an error (although it should be throwing a `std::system_error` rather than a bare `std::runtime_error`), and that's the only way this can error?